src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -85,9 +85,9 @@ public function get(string $key, callable $callback, ?float $beta = null, ?array
         }
         if (!isset($this->keys[$key])) {
             get_from_pool:
-            if ($this->pool instanceof CacheInterface) {
-                return $this->pool->get($key, $callback, $beta, $metadata);
-            }
+                if ($this->pool instanceof CacheInterface) {
+                    return $this->pool->get($key, $callback, $beta, $metadata);
+                }
 
             return $this->doGet($this->pool, $key, $callback, $beta, $metadata);
```

Comment:
Sth went wrong here

either we put indent to whole section for `get_from_pool:` label, or to none of it